internal/datanode/metacache/bloom_filter_set.go

```diff
@@ -59,7 +76,7 @@ func (bfs *BloomFilterSet) UpdatePKRange(ids storage.FieldData) error {
 
 	if bfs.current == nil {
 		bfs.current = &storage.PkStatistics{
-			PkFilter: bloom.NewWithEstimates(paramtable.Get().CommonCfg.BloomFilterSize.GetAsUint(),
+			PkFilter: bloom.NewWithEstimates(bfs.batchSize,
```

Comment:
if the estimated batchSize is smaller than the real batch size, would PkFilter able to handle it ? @congqixia

Reply:
that's fine. the side effect will be large false positive rate than expected